### DISCUSSION TITLE
add Zweckverband Abfallwirtschaft Oberes Elbtal?

### DISCUSSION THREAD

**Discussion by Schmarni** [mampfes/hacs_waste_collection_schedule#778] [Ideas]
Hello,
can you please add source for Zweckverband Abfallwirtschaft Oberes Elbtal in Germany?
https://www.zaoe.de/abfallkalender/entsorgungstermine/
Thank you very much :-)

**Comment by 5ila5** [comment]
zaoe.de provides a ICALS subscription URL you should be able to use the [ICS/ICAL source](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md) with the URL you get when you click on (als iCal-Abonnement)

**Comment by 5ila5** [comment]
officially documented with #865
